python/lsst/obs/base/cameraMapper.py

```diff
@@ -1165,14 +1165,16 @@ def _initWriteRecipes(self):
         recipesFile = os.path.join(getPackageDir("obs_base"), "policy", "writeRecipes.yaml")
         recipes = dafPersist.Policy(recipesFile)
         supplementsFile = os.path.join(self.getPackageDir(), "policy", "writeRecipes.yaml")
+        validationMenu = {'FitsStorage': validateRecipeFitsStorage,}
         if os.path.exists(supplementsFile) and supplementsFile != recipesFile:
             supplements = dafPersist.Policy(supplementsFile)
             # Don't allow overrides, only supplements
-            intersection = set(recipes.names()).intersection(set(supplements.names()))
-            if intersection:
-                raise RuntimeError("Recipes provided in %s may not override those in %s: %s" %
-                                   (supplementsFile, recipesFile, intersection))
-            recipes.update(overrides)
+            for entry in validationMenu:
+                intersection = set(recipes[entry].names()).intersection(set(supplements.names()))
+                if intersection:
+                    raise RuntimeError("Recipes provided in %s section %s may not override those in %s: %s" %
+                                       (supplementsFile, entry, recipesFile, intersection))
+            recipes.update(supplements)
 
         self._writeRecipes = {}
         validationMenu = {'FitsStorage': validateRecipeFitsStorage, }
```

Comment:
Can you delete this line now that you have inserted a copy before `if os.path.exists...`?